components/inputs/input-date-time.js

```diff
@@ -220,6 +220,10 @@ class InputDateTime extends FormElementMixin(LocalizeCoreElement(RtlMixin(LitEle
 			await this.updateComplete;
 			this._handleInputTimeFocus();
 		}
+		this.dispatchEvent(new CustomEvent(
+			'd2l-input-date-time-dropdown-toggle',
+			{ bubbles: true, composed: false, detail: { opened: e.detail.opened } }
```

Comment:
Does this need to `bubble`? It might be nice to force people to handle it on the `d2l-input-date-time` itself